app/javascript/components/ActionItemDashboard.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
```

Comment:
Also probably best to put this component in its own directory like you did with ActionItemCard